extension/manifest.json

```diff
@@ -1,32 +1,40 @@
 {
   "manifest_version": 2,
-  "name": "speak-to-me",
+  "name": "Voice Fill",
   "version": "1.2.1",
+  "description": "Adds voice input to popular search pages in Firefox. Learn more about Voice Fill at https://testpilot.firefox.com",
   "developer": {
     "name": "Fabrice Desré",
```

Comment:
Although Fabrice did work on this project early on, Andre has done the majority of the work (development, debugging, cross coordination and working with the Test Pilot team at large).  I'd like to have him listed there as well.